pages/outputs/[id].jsx

```diff
@@ -0,0 +1,149 @@
+import React from 'react'
+import Head from 'next/head'
+import { Header } from '@oacore/design'
+
+import request from 'api'
+import { fetchMetadata, fetchSimilarTo, fetchCitations } from 'api/outputs'
+import { useStore } from 'store'
+import Meta from 'modules/meta'
+import Template from 'templates/output'
+
+const LOCALE = 'en-GB'
+const CITATION_STYLES = ['apa', 'bibtex']
+
+// Needed for clear development of the data retrieval
+// At the time of development the API was unstable
+const log = (...args) => {
+  if (process.env.NODE_ENV !== 'production')
+    // eslint-disable-next-line no-console
+    console.log(...args)
+}
+
+export async function getStaticProps({ params: routeParams }) {
```

Comment:
`getStaticProps` and `getServerSideProps` are 2 interchangeable functions in the Next.js. Both receive parameters from the route, `id` in our case and use it to retrieve data from the API.